cocotb/handle.py

```diff
@@ -139,7 +139,7 @@ def __setattr__(self, name, value):
     def __getattr__(self, name):
         if name in self._compat_mapping:
             if name not in _deprecation_warned:
-                warnings.warn("Use of %s attribute is deprecated" % name)
+                warnings.warn("Use of attribute '%s' is deprecated, use '%s' instead" % (name, self._compat_mapping[name]))
                 _deprecation_warned[name] = True
```

Comment:
Better to use `%r` instead of `'%s'`, it adds quotes for you

Reply:
Done, thanks.